### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by Tibase**
Modify Run command

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
@rande I think this should be corrected instead: https://github.com/sonata-project/sandbox/blob/5bc789200ddb6e5f00804edab72e89df43ce8526/composer.json#L137

**Comment by OskarStark** [comment]
ping @Soullivaneuh

**Comment by OskarStark** [comment]
ping @rande

**Comment by OskarStark** [comment]
ping @core23

**Comment by core23** [comment]
Never used the sandbox, but looks okay to me

**Comment by core23** [comment]
Looks good, but can you change the commit and PR title @Tibase

**Comment by OskarStark** [comment]
@Soullivaneuh can you please give us a finale review? If everything is fine, we can squash merge this and change the commit title

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Get composer:
 
 Run the following command for the 2.4 develop branch:
 
-    php composer.phar create-project sonata-project/sandbox:2.4.x-dev
+    php composer.phar create-project sonata-project/sandbox:dev-2.4-develop
```
Comment (greg0ire) [inline README.md]: The fact we need to do that shows that maybe we should rethink the way this branch is named.

Comment (OskarStark) [inline README.md]: or create releases

Comment (greg0ire) [inline README.md]: or both

**Comment by Soullivaneuh** [comment]
This is not relevant anymore.

Will be refactored soon.

**Pull request closed without merging**